index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bootcamp: Ticket 1</title>
+    <link rel="stylesheet" href="main.css"/>
+
+    <style>
+        @font-face{
+        font-family: 'Karla Regular';
+        font-style: normal;
+        font-weight: normal;
+        src: local('Karla Regular'), url('fonts/karla-regular.ttf') format('truetype');
+        }
+
+        @font-face{
+        font-family: 'Karla Bold';
+        font-style: normal;
+        font-weight: bold;
+        src: local('Karla Bold'), url('fonts/karla-bold.ttf') format('truetype');
+        }
+
+        @font-face{
+        font-family: 'Juana';
+        font-style: normal;
+        font-weight: normal;
+        src: local('Juana'), url('fonts/juana.otf') format('opentype');
+        }
+
+        @font-face{
+        font-family: 'Tropical Summer';
+        font-style: normal;
+        font-weight: normal;
+        src: local('Tropical Summer'), url('fonts/tropical-summer.otf') format('opentype');
+        }
+        
+    </style>
+    
+</head>
+
+<body>
+
+    <header>
+        <div class="heading">
+            <div class="nav-column search" id="search">
+                <!-- Sorry not sure this is what you wanted! -->
+                <a href="search.html"> 
+                    <img class="search-icon" src="img/search.svg"/>
+                </a>
+            </div>
+            <img class="nav-column tws-logo" src="img/tws-logo.png"/>

```

Comment:
Logo should always be a link to the homepage ('/')

Reply:
I can also see the logo is poor quality - when exporting from Figma you can choose option 'x2' for size